### PR TITLE
Jeren ug update Jer En's features

### PR DESCRIPTION
**Pull request by jerrrren**
Update UG for Jer En's portion and update About US

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -367,87 +367,82 @@ Examples:
 
 ### Adding an event: `jeren`
```
Comment (kennethk-1201) [inline docs/UserGuide.md:368]: my bad didn't say properly, remember to replace the `jeren` section with `<command-name>`:
eg. Adding an event: `addEvent`

**Review by kennethk-1201 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (kennethk-1201) [reply docs/UserGuide.md:368]: same with all the subsequent lines containing `jeren`

Comment (jerrrren) [reply docs/UserGuide.md]: changed

**Review by kennethk-1201 — APPROVED** [review]
lgtm

**Pull request merged**